pytuya/__init__.py

```diff
@@ -27,16 +27,14 @@
     Crypto = AES = None
     import pyaes  # https://github.com/ricmoo/pyaes
 
+from pytuya.const import __version__
```

Comment:
Looks like this file is missing from the PR/commit

Reply:
Oh sorry, missing `git add`... This is now fixed.

Reply:
That was fast! 👍